config.mk

```diff
@@ -6,6 +6,7 @@ DATADIR ?= ${PREFIX}/share
 # around for backwards compatibility
 MANPREFIX ?= ${DATADIR}/man
 MANDIR ?= ${MANPREFIX}
+CFLAGS_CI ?=
```

Comment:
I would call it `EXTRA_CFLAGS` and document it somewhere. It might be useful for other things as well.

Reply:
I've renamed it to `EXTRA_CFLAGS`.